cmd/res/example/configuration.toml

```diff
@@ -68,3 +68,24 @@ Level = "DEBUG"
        Password = "public"
        Topic = "CommandTopic"
 
+# Driver configs
+[Driver]
+IncomingSchema = "tcp"
+IncomingHost = "0.0.0.0"
+IncomingPort = "1883"
+IncomingUser = "admin"
+IncomingPassword = "public"
+IncomingQos = "0"
+IncomingKeepAlive = "3600"
+IncomingClientId = "IncomingDataSubscriber"
+IncomingTopic = "DataTopic"
+
```

Comment:
why empty line?

Reply:
I want to separate  Incoming and Response settings